contents/docs/apps/build/reference.md

```diff
@@ -228,37 +272,21 @@ export function setupPlugin({ attachments, global }) {
 }
 ```
 
-If you throw a `RetryError` (imported from `@posthog/plugin-scaffold`) in your `setupPlugin` function, PostHog will retry the initialization up to 10 times within an hour before disabling the plugin. Errors other than `RetryError` cause the app to be disabled automatically.
+`setupPlugin` can be retried up to 5 times (first retry after 5 s, then 10 s after that, 20 s, 40 s, lastly 80 s) by throwing [`RetryError`](#maximizing-reliability-with-retryerror). Attempting to retry more than 5 times disables the plugin. The plugin is disabled immediately if any error other than `RetryError` is thrown in `setupPlugin`.
 
-`RetryError` should be used to indicate an error that is dependent on an external service, meaning that retrying it may actually lead to a different outcome (success). If you throw a `RetryError` because parsing a config option fail, it will never actually succeed.
+On PostHog Cloud and [email-enabled](/docs/self-host/configure/email) instances of PostHog, project members are notified by email of the plugin being disabled automatically. This is to ensure that action is taken if the plugin is important for data integrity.
 
-Example:
+## `teardownPlugin` function
```

Comment:
I think we forgot about `teardownPlugin`. Here I'm assuming we won't add retries to it.